includes/rest-api/class-sensei-rest-api-question-helpers-trait.php

```diff
@@ -88,35 +88,33 @@ private function save_category_question( $question ) {
 			$question['options'] = [];
 		}
 
-		$category             = null;
 		$question_id          = isset( $question['id'] ) ? (int) $question['id'] : null;
 		$question_number      = (int) $question['options']['number'];
 		$question_category_id = (int) $question['options']['category'];
+		$question_category    = false;
 
 		if ( $question_category_id ) {
 			$question_category = get_term( $question_category_id, 'question-category' );
 		}
 
 		if ( ! $question_category || is_wp_error( $question_category ) ) {
-			return new WP_Error( 'sensei_lesson_quiz_question_invalid_category', esc_html__( 'Invalid question category selected', 'sensei-lms' ), $question_id );
+			return new WP_Error( 'sensei_lesson_quiz_question_invalid_category', esc_html__( 'Invalid question category selected.', 'sensei-lms' ), $question_id );
 		}
 
 		if ( ! $question_number ) {
 			$question_number = 1;
 		}
 
-		$category = get_term( $question_category, 'question-category' );
-
 		// translators: Placeholders are the question number and the question category name.
-		$post_title = sprintf( esc_html__( '%1$s Question(s) from %2$s', 'sensei-lms' ), $question_number, $category->name );
+		$post_title = sprintf( esc_html__( '%1$s Question(s) from %2$s', 'sensei-lms' ), $question_number, $question_category->name );
 
 		$post_args = [
 			'ID'          => $question_id,
 			'post_title'  => $post_title,
 			'post_status' => 'publish',
 			'post_type'   => 'multiple_question',
 			'meta_input'  => [
-				'category' => $category->term_id,
+				'category' => $question_category ? $question_category->term_id : '',
```

Comment:
No need for a check here because of the early return earlier. 

```suggestion
				'category' => $question_category->term_id,
```

Reply:
Ahh yeah, that was from when I was trying to make invalid questions save.